test/take.js

```diff
@@ -39,4 +39,19 @@ describe('take', function() {
     eq(R.into([], R.take(0), [1, 2, 3]), []);
   });
 
+  it('steps correct number of times', function() {
+    var takeXTimes = function(x) {
+      var counter = 0;
+      R.into(
+        [],
+        R.compose(R.map(function(x) { counter += 1; return x; }), R.take(x)),
```

Comment:
i understand that, but what bothers me is modeling undesirable practices in the test suite. the test suite should be examples of how the library should be used. It makes sense to me to write idiomatic code for the test, and let the *test framework* handle the side-effects.